lib/twilio/audiohelper.ts

```diff
@@ -275,6 +275,16 @@ class AudioHelper extends EventEmitter {
     if (isEnumerationSupported) {
       this._initializeEnumeration();
     }
+
+    navigator.permissions.query({ name: 'microphone' as PermissionName }).then((microphonePermissionStatus) => {
```

Comment:
Do we need to cast `'microphone' as PermissionName` here? I don't believe so from my testing unless I'm missing something.

Reply:
My VScode is giving my a type error, but it looks like it compiles fine without the typecast 👍